### DISCUSSION TITLE
How to change color of "userChromeJS Manager" Add-on?

### DISCUSSION THREAD

**Discussion by rediffusion** [xiaoxiaoflood/firefox-scripts#139] [General Support]
I don't like that it's blue color!
( ఠ ͟ʖ ఠ)

![userChromeJS Manager](https://user-images.githubusercontent.com/30071027/169031676-e9da87f6-bfe6-4a34-9b1f-2396f0b08ace.png)

**Comment by xiaoxiaoflood** [comment]
Change `blue` to whatever you want.

https://github.com/xiaoxiaoflood/firefox-scripts/blob/77f56676dc6686af15827f9d51c1e4d3e0989c2a/chrome/rebuild_userChrome.uc.js#L420-L428
